tarsier-service/service/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Use the official Python 3.12 image as the base
+FROM python:3.12-slim
```

Comment:
nit: not sure if 3.12 is more stable now but it's had some issues in the past. 3.11 might be the safer choice